config.go

```diff
@@ -2,15 +2,24 @@ package main
 
 import (
 	"bufio"
+	_ "embed"
 	"fmt"
 	"os"
+	"path/filepath"
 	"strings"
 	"text/tabwriter"
 
+	"github.com/bitrise-io/go-utils/fileutil"
 	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
 	testing "google.golang.org/api/testing/v1"
 )
 
+// This embedded basic application was created based on the https://github.com/tothszabi/BasicApp repository.
+//
+//go:embed resources/app.apk
```

Comment:
nit: Maybe we should give it a better name than `app.apk` to make it obvious what this is used for